modules/ROOT/pages/_partials/sync/js-client-advanced-topics.adoc

```diff
@@ -77,3 +77,129 @@ The `OptimisticResponse` feature and the <<#sync-client-offline-queue-listener,
 //tag::excludeDownstream[]
 See the xref:showcase-apps.adoc[Showcase Apps section] for more information.
 //end::excludeDownstream[]
+
+== Cache Update Helpers
+
+{sync-client} provides an out of the box solution for managing updates to your application's cache.
```

Comment:
what do you need to begin with.
This suggests that the developer already has cache code, is that true?

Reply:
Yes thats correct. They will have a cache already based on the default config. Without this change they will need to manage updates to that cache themselves.